python/pipeline/pupil.py

```diff
@@ -915,42 +879,185 @@ def make(self, key):
             config['orig_video_path'] = video_path
 
             # find croppoing coords
-            cropped_coords = (Tracking.Deeplabcut & key).fetch1('cropped_x0','cropped_x1','cropped_y0','cropped_y1')
+            cropped_coords = (Tracking.Deeplabcut & key).fetch1(
+                'cropped_x0', 'cropped_x1', 'cropped_y0', 'cropped_y1')
 
             config['cropped_coords'] = cropped_coords
 
-            pupil_fit = DLC_tools.PupilFitting(config=config, bodyparts='all', cropped=True)
+            pupil_fit = DLC_tools.DeeplabcutPupilFitting(
+                config=config, bodyparts='all', cropped=True)
 
             for frame_num in tqdm(range(nframes)):
 
                 fit_dict = pupil_fit.fitted_core(frame_num=frame_num)
 
                 # circle info
-                center=fit_dict['circle_fit']['center']
-                radius=fit_dict['circle_fit']['radius']
-                visible_portion=fit_dict['circle_visible']['visible_portion']
+                center = fit_dict['circle_fit']['center']
+                radius = fit_dict['circle_fit']['radius']
+                visible_portion = fit_dict['circle_visible']['visible_portion']
 
-                data_circle.append([*common_entry, frame_num, center, radius, visible_portion])
+                data_circle.append(
+                    [center, radius, visible_portion])
 
                 # ellipse info
-                center=fit_dict['ellipse_fit']['center']
-                major_radius=fit_dict['ellipse_fit']['major_radius']
-                minor_radius=fit_dict['ellipse_fit']['minor_radius']
-                rotation_angle=fit_dict['ellipse_fit']['rotation_angle']
-                visible_portion=fit_dict['ellipse_visible']['visible_portion']
-
-                data_ellipse.append([*common_entry, frame_num, center, 
-                                            major_radius, minor_radius, 
-                                            rotation_angle, visible_portion])
-        
+                center = fit_dict['ellipse_fit']['center']
+                major_radius = fit_dict['ellipse_fit']['major_radius']
+                minor_radius = fit_dict['ellipse_fit']['minor_radius']
+                rotation_angle = fit_dict['ellipse_fit']['rotation_angle']
+                visible_portion = fit_dict['ellipse_visible']['visible_portion']
+
+                data_ellipse.append([center,
```

Comment:
major radius can be put in the line above

Reply:
updated